ViaChristopheH/Questioner_Repos
# Linting

#### What does this PR do?
- install eslint
-update UI
-update APIs
#### Description of Task to be completed?
- for code clearing
- for improving the user experience
-for the test efficiency
#### How should this be manually tested?
-should be run through the command line interface
-should be tested via the browser
-should be run via postman, travisCI and heruko
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163164718
https://www.pivotaltracker.com/story/show/163164771
#### Screenshots (if appropriate)
-
